tests/utils.js

```diff
@@ -0,0 +1,45 @@
+var assert = require("assert");
+
+
+var DATASET_HEADER_MAP = {
+  conditions: "Condition",
+  damageDeckCore: "DamageDeckCore",
+  damageDeckCoreTfa: "DamageDeckCore",
+  pilots: "Pilot",
+  ships: "Ship",
+  sources: "Source",
+  upgrades: "Upgrade",
+};
+
+function buildDataHeader(data, dataKey ,index) {
+  var dataHeader;
+  var header = DATASET_HEADER_MAP[dataKey];
+  var model = data[dataKey][index];
+
+  if (model.id && model.name) {
```

Comment:
If `id` is `0` this first part will evaluate to `false`.

Use `if ('id' in model && model.name) {` instead.